phpunit.xml.dist

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't agree with that at all. This project isn't some base project for people to base other projects off of like a framework. It's a single, definitive project that should have its own standards and not allow user's choice as the default. If someone wants to use their own settings they can modify the phpunit.xml in their own local checkout.